scheduler/clients/javascript/tests/integration/integration.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { Calendar, INettuClient, User, CalendarEvent } from "../../lib";
+import { setupAccount } from "../helpers/fixtures";
+
+// This test suite is testing the specifications for our use cases
+
+describe("Integration", () => {
+  let client: INettuClient | undefined;
+  let accountId: string | undefined;
+
+  let user1: User | undefined;
+  let user1Calendar1: Calendar | undefined;
+  let user1Calendar1Event1: CalendarEvent | undefined;
+  let user1Calendar2: Calendar | undefined;
+  // let user1Calendar2Event1: CalendarEvent | undefined;
+
+  beforeAll(async () => {
+    const account = await setupAccount();
+
+    accountId = account.accountId;
+    client = account.client;
+  });
+
+  describe("Basic functionalities", () => {
+    it("should create a user", async () => {
+      // TODO: we cannot provide an ID for the user
+      // For our use case, we would like to provide our own ID
+      // To be adapted
+      const res = await client?.user.create();
+      if (!res?.data) {
+        throw new Error("User not created");
```

Comment:
[question] I've seen you do this pattern of throwing an error in some tests, how come, instead of just doing an `expect`?

Reply:
`expect` unfortunately doesn't seem to tell TS that the response is safe/not undefined

<img width="598" alt="image" src="https://github.com/user-attachments/assets/0a0d3a60-8b79-4c99-a6d8-f37cd8d15003">
 
Technically, adding `?` everywhere works though

<img width="313" alt="image" src="https://github.com/user-attachments/assets/07af87c7-2493-44df-8b68-3b11cd2619fa">

We could do that, I don't mind personally

Reply:
Nah I like your approach. Thanks!